packages/core/src/controllers/BlockchainApiController.ts

```diff
@@ -304,7 +306,10 @@ export const BlockchainApiController = {
 
   async reverseLookupEnsName({ address }: { address: string }) {
     return state.api.get<BlockchainApiLookupEnsName[]>({
-      path: `/v1/profile/reverse/${address}?projectId=${OptionsController.state.projectId}`
+      path: `/v1/profile/reverse/${address}?projectId=${OptionsController.state.projectId}`,
+      params: {
+        sender: AccountController.state.address
+      }
```

Comment:
seems like we can sen `projectId` also as a param here

Reply:
updated all occurrences of hardocded params